forkpi/records/models.py

```diff
@@ -61,3 +61,21 @@ class Option(Model):
 class Door(Model):
     name = TextField(default="", unique=True)
     serial = TextField(unique=True)
+
+
+class AppConfigManager(Manager):
+    def get_singleton(self):
+        obj, created = self.get_or_create(pk=1)
+        return obj
+
+
+class AppConfig(Model):
+    global_pin = TextField(default="", db_index=True, null=True, blank=True)
```

Comment:
You don't need index on global_pin now.
```suggestion
    global_pin = TextField(default="", null=True, blank=True)
```